src/cptdl/DLcolors.py

```diff
@@ -0,0 +1,260 @@
+import matplotlib.cm as cm
+from matplotlib.colors import LinearSegmentedColormap
+import matplotlib.pyplot as plt
+#
+from typing import Tuple, List, Literal, Optional, Union, Callable, Iterable as Iterable
+from typing import NamedTuple
+import numpy as np
+
+
+def graforientation(Horizontal,Vertical):
+    if Horizontal>=Vertical:
+        return 'horizontal'
+    else:
+        return 'vertical'
+
+def MinMaxBar(Min, Max):
```

Comment:
Both function name and variables names should be lowercase and snake_case (`min_max_bar`, `min`, `max`).